app/workers/articles/bust_multiple_caches_worker.rb

```diff
@@ -0,0 +1,13 @@
+module Articles
+  class BustMultipleCachesWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :default
+
+    def perform(article_ids, cache_buster = CacheBuster)
```

Comment:
Although it's not merged yet, you can take a look at this initial PR - https://github.com/thepracticaldev/dev.to/pull/5302/files 

`cache_buster` should become a string, so that we are sure only simple objects like integers and strings are serialized